### DISCUSSION TITLE
EXPECT_CALL equivalent

### DISCUSSION THREAD

**Discussion by reymalahay** [bxparks/AUnit#119] [Q&A]
hi there,

does `AUnit` have a `EXPECT_CALL` function like `googlemock`? or is this functionality accessible via the `GoogleTest Adapter`?

what i need is when i pass in a mock object to my `sut`, i want to verify that certain functions where invoked and if so, how many times.

please advise.

thanks in advance,
rey malahay

**Comment by bxparks** [comment]
As explained in another thread, you are flooding me with Issues and Discussions: 14 tickets in 3 weeks. Even just reading them takes time and energy, to figure out whether it is a valid Issue, or whether it belongs in a Discussion. Some of your posts have been duplicates. Some have been filed in the wrong project. I just don't have the time and energy for your tickets. So I am going to close all of them, because I won't be able to get to them.
